pkg/sdk/poc/generator/show_by_id_filtering.go

```diff
@@ -14,32 +13,68 @@ const (
 	// Enables filtering with: In
 	// Based on the identifier Kind
 	ShowByIDInFiltering
-	// Enables filtering with: ExtendedIn
+	// Enables filtering with: In
 	// Based on the identifier Kind
 	ShowByIDExtendedInFiltering
+	// Enables filtering with: Limit
+	ShowByIDLimitFiltering
 )
 
+type ShowByIDFiltering interface {
+	WithFiltering() string
+}
+
+type showByIDFilter struct {
+	Name string
+	Kind string
+	Args string
+}
+
+func (s *showByIDFilter) WithFiltering() string {
+	return fmt.Sprintf("With%s(%s{%s})", s.Name, s.Kind, s.Args)
+}
+
+var filteringMapping = map[ShowByIDFilteringKind]func(string) ShowByIDFiltering{
+	ShowByIDLikeFiltering:       newShowByIDLikeFiltering,
+	ShowByIDInFiltering:         newShowByIDInFiltering,
+	ShowByIDExtendedInFiltering: newShowByIDExtendedInFiltering,
+	ShowByIDLimitFiltering:      newShowByIDLimitFiltering,
+}
+
+func newShowByIDFiltering(name, kind, args string, identifierKind *string) ShowByIDFiltering {
```

Comment:
How about using a variadic function?

Reply:
will be done by formatting arguments in the create function (as discussed verbally). This way the identifier kind is removed from `newShowByIDFiltering()` function and there is no need for variadic anymore.